spec/helper/revenuecat_internal_helper_spec.rb

```diff
@@ -74,6 +74,9 @@
   end
 
   describe '.auto_generate_changelog' do
+    before(:each) do
+      Fastlane::Helper::RevenuecatInternalHelper.cleanup_caches
+    end
```

Comment:
Yeah... I don't know... Honestly, using class methods is not my preferred choice, but since everything in fastlane seems to be done with them, I followed the convention. I think this works for now...